### PR TITLE
UI For Library application

### PR DESCRIPTION
**Pull request by Itswali**
### IN THIS PR.

-    Build interactive console apps.
-    Run a program using the command line.

- `UI Screen Shots`
 
image

### CODE REVIEW

File: modules/student.rb [hunk 1]
```diff
@@ -1,11 +1,20 @@
-require '../ibrary'
+require_relative '../person'
+require_relative './classroom'
+
 class Student < Person
-  def initialize(name, age, parent_permission, classroom)
-    super(name, age, parent_permission)
+  attr_reader :classroom
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
```
Comment (codecaiine) [inline modules/student.rb:8]: - [ ] There are some errors in your app this in your app as you can see in the screenshot below ⬇️  Please be sure to fix them in order to make your app able to create a student and also a rental to make it works properly 👍 

Capture

File: modules/teacher.rb [hunk 2]
```diff
@@ -1,11 +1,11 @@
-require '../library'
+require_relative '../person'
 class Teacher < Person
-  def initialize(name, age, parent_permission, specialization)
-    super(name, age, parent_permission)
+  def initialize(age, specialization, name = 'unknown')
```
Comment (codecaiine) [inline modules/teacher.rb]: - [ ] There are some errors in your app this in your app as you can see in the screenshot below ⬇️  Please be sure to fix them in order to make your app able to create a teacher and also a rental to make it works properly  👍 

Capture

**Review by codecaiine — CHANGES_REQUESTED** [review]
Hi @Itswali , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Great UI implementation ✔️
-  No rubocop errors ✔️

## Required Changes ♻️
- [ ] It's very important to make you make your app behaves in the same way as the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ) make sure that we are able to see all the thing listed in works as expected without errors 👍 .
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hiromataba — APPROVED** [review]
# Status: Approved :tada: :tada: :tada: 
Hi @Itswali    :wave: :wave: 

200w  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**